KitePrintSDK/src/main/java/ly/kite/checkout/PaymentActivity.java

```diff
@@ -161,15 +202,25 @@ public boolean onMenuItemSelected(int featureId, MenuItem item) {
     }
 
     public void onButtonPayWithPayPalClicked(View view) {
-        PayPalPayment payment = new PayPalPayment(printOrder.getCost(printOrder.getCurrencyCode()), printOrder.getCurrencyCode(), "Product");
-        Intent intent = new Intent(this, com.paypal.android.sdk.payments.PaymentActivity.class);
-        intent.putExtra(com.paypal.android.sdk.payments.PaymentActivity.EXTRA_PAYPAL_ENVIRONMENT, printEnvironment.getPayPalEnvironment());
-        intent.putExtra(com.paypal.android.sdk.payments.PaymentActivity.EXTRA_CLIENT_ID, printEnvironment.getPayPalClientId());
-        intent.putExtra(com.paypal.android.sdk.payments.PaymentActivity.EXTRA_PAYER_ID, "<someuser@somedomain.com>");
-        intent.putExtra(com.paypal.android.sdk.payments.PaymentActivity.EXTRA_RECEIVER_EMAIL, printEnvironment.getPayPalReceiverEmail());
-        intent.putExtra(com.paypal.android.sdk.payments.PaymentActivity.EXTRA_PAYMENT, payment);
-        intent.putExtra(com.paypal.android.sdk.payments.PaymentActivity.EXTRA_SKIP_CREDIT_CARD, true);
-        startActivityForResult(intent, REQUEST_CODE_PAYPAL);
+
+//        intent.putExtra(com.paypal.android.sdk.payments.PaymentActivity.EXTRA_PAYPAL_ENVIRONMENT, printEnvironment.getPayPalEnvironment());
+//        intent.putExtra(com.paypal.android.sdk.payments.PaymentActivity.EXTRA_CLIENT_ID, printEnvironment.getPayPalClientId());
+//        intent.putExtra(com.paypal.android.sdk.payments.PaymentActivity.EXTRA_PAYER_ID, "<someuser@somedomain.com>");
```

Comment:
I've wondered that too - especially as I noticed that if you go the 
PayPal route (as opposed to the credit card route) you also get a credit 
card option.

On 21/07/15 14:51, Deon Botha wrote:
>
> In KitePrintSDK/src/main/java/ly/kite/checkout/PaymentActivity.java 
> <https://github.com/OceanLabs/Android-Print-SDK/pull/9#discussion_r35103386>:
>
> > @@ -161,15 +202,25 @@ public boolean onMenuItemSelected(int featureId, MenuItem item) {
> >      }
> >
> >      public void onButtonPayWithPayPalClicked(View view) {
> > -        PayPalPayment payment = new PayPalPayment(printOrder.getCost(printOrder.getCurrencyCode()), printOrder.getCurrencyCode(), "Product");
> > -        Intent intent = new Intent(this, com.paypal.android.sdk.payments.PaymentActivity.class);
> > -        intent.putExtra(com.paypal.android.sdk.payments.PaymentActivity.EXTRA_PAYPAL_ENVIRONMENT, printEnvironment.getPayPalEnvironment());
> > -        intent.putExtra(com.paypal.android.sdk.payments.PaymentActivity.EXTRA_CLIENT_ID, printEnvironment.getPayPalClientId());
> > -        intent.putExtra(com.paypal.android.sdk.payments.PaymentActivity.EXTRA_PAYER_ID, "<someuser@somedomain.com>");
> > -        intent.putExtra(com.paypal.android.sdk.payments.PaymentActivity.EXTRA_RECEIVER_EMAIL, printEnvironment.getPayPalReceiverEmail());
> > -        intent.putExtra(com.paypal.android.sdk.payments.PaymentActivity.EXTRA_PAYMENT, payment);
> > -        intent.putExtra(com.paypal.android.sdk.payments.PaymentActivity.EXTRA_SKIP_CREDIT_CARD, true);
> > -        startActivityForResult(intent, REQUEST_CODE_PAYPAL);
> > +
> > +//        intent.putExtra(com.paypal.android.sdk.payments.PaymentActivity.EXTRA_PAYPAL_ENVIRONMENT, printEnvironment.getPayPalEnvironment());
> > +//        intent.putExtra(com.paypal.android.sdk.payments.PaymentActivity.EXTRA_CLIENT_ID, printEnvironment.getPayPalClientId());
> > +//        intent.putExtra(com.paypal.android.sdk.payments.PaymentActivity.EXTRA_PAYER_ID, "<someuser@somedomain.com>");
>
> guess most of this is redundant comments too now -- although I'd be 
> interest to know if the latest PayPal SDK does have an equivalent too 
> EXTRA_SKIP_CREDIT_CARD?
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/OceanLabs/Android-Print-SDK/pull/9/files#r35103386>.
>